sql/outgoing_request_cache.sql

```diff
@@ -19,4 +19,4 @@ CREATE TABLE `api_cache`(
     `content` text NOT NULL,
     PRIMARY KEY (id),
     FOREIGN KEY (config_id) REFERENCES outgoing_request_cache_config(id)
-)AUTO_INCREMENT=8 DEFAULT CHARSET=latin1;
+)AUTO_INCREMENT=8 DEFAULT CHARSET=utf8;
```

Comment:
Some of the symbols in some Printful API responses are UTF-8. One of the errors lists `\\xE2\\x80\\xB3\\xE2\\x80\\x93...` as an issue. It might actually be a different charset, but given that it's not throwing errors over it anymore, I think UTF-8 will be fine.

Reply:
HTTP is always ISO-8859-1 charset, we can transform that into whatever client side charset we feel like. Latin1 should be fine, but if not we can update it to utf-8 if there's a specific problem.

Reply:
The issue was inserting response contents into the table. MySQL was giving an "incorrect string value" error for some products. The character I mentioned above is the [double prime character](https://www.compart.com/en/unicode/U+2033) which I don't think is present in ISO-8859-1. It's in the product description for every hat product

Reply:
Interesting, the curl library must be converting to UTF-8 instead of the standard ISO-8859-1